src/lang/isBoolean.js

```diff
@@ -0,0 +1,8 @@
+const isBoolean = (value) => {
+  if (typeof value === 'boolean') {
+    return true;
+  }
+    return false;
+}
```

Comment:
This good code, good job,  but I just want to pass along the info can be shortened to:
```
return (typeof value === 'boolean')
```